lsp/client/visualStudio/IdesLspPoc/ContentDefinitions/JsonContentType.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.VisualStudio.LanguageServer.Client;
+using Microsoft.VisualStudio.Utilities;
+using System.ComponentModel.Composition;
+
+namespace IdesLspPoc.ContentDefinitions
+{
+    // This is used to associate our language server with JSON files
+    // JSON handling is sort of reserved by VS, this is the only way I found that activates our LSP and uses it with .json files
+    // See FAQ: https://learn.microsoft.com/en-us/visualstudio/extensibility/adding-an-lsp-extension?view=vs-2022#faq
+    public static class JsonContentType
+    {
+        // This also becomes the languageId provided to the language server
+        public const string ContentTypeName = "placeholder-test-json";
```

Comment:
This will get a better/clearer name in the future?

Reply:
Yes -- this is illustrative of how we will likely have to define some language ids if we want to codify logic using language ids in the language server.